lib/galaxy_test/selenium/test_custom_builds.py

```diff
@@ -55,11 +55,11 @@ def add_custom_build(self, build_name, build_key):
     def delete_custom_build(self, build_name):
         delete_button = None
         grid = self.wait_for_selector("table.grid > tbody")
-        for row in grid.find_elements_by_tag_name("tr"):
-            td = row.find_elements_by_tag_name("td")
+        for row in grid.find_elements(self.by.CSS_SELECTOR, "tr"):
+            td = row.find_elements(self.by.CSS_SELECTOR, "td")
```

Comment:
Shouldn't these be:

```suggestion
        for row in grid.find_elements(self.by.TAG_NAME, "tr"):
            td = row.find_elements(self.by.TAG_NAME, "td")
```

?